packages/cardpay-subgraph/subgraph-template.yaml

```diff
@@ -545,6 +545,131 @@ dataSources:
           handler: handleTransfer
       file: ./src/mappings/token.ts
 
+  - kind: ethereum/contract
+    name: RewardManager
+    network: {NETWORK}
+    source:
+      address: "{REWARD_MANAGER_ADDRESS}"
+      abi: RewardManager
+      startBlock: {CARDPAY_GENESIS_BLOCK}
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.4
+      language: wasm/assemblyscript
+      entities:
+        - RewardSafe
+      abis:
+        - name: RewardManager
+          file: ./abis/generated/RewardManager.json
+        - name: ERC20
+          file: ./abis/ERC20.json
+        - name: ERC20SymbolBytes
+          file: ./abis/ERC20SymbolBytes.json
+        - name: ERC20NameBytes
+          file: ./abis/ERC20NameBytes.json
+        - name: GnosisSafe
+          file: ./abis/GnosisSafe.json
+      eventHandlers:
+        - event: RewardeeRegistered(address,address,address)
+          handler: handleRewardeeRegistration
+      file: ./src/mappings/reward-manager.ts
+
+  - kind: ethereum/contract
+    name: RegisterRewardProgram
+    network: {NETWORK}
+    source:
+      address: "{REGISTER_REWARD_PROGRAM_HANDLER_ADDRESS}"
+      abi: RegisterRewardProgramHandler
+      startBlock: {CARDPAY_GENESIS_BLOCK}
```

Comment:
I'd recommend a higher start block here to make life easier for indexing (this was introduced in 0.7.0 or 0.8.0, i can't remember which